sdk/graphrbac/graph/README.md

```diff
@@ -1,6 +1,14 @@
 ## Azure GraphRbacManagementClient SDK for JavaScript
 
-This package contains an isomorphic SDK for GraphRbacManagementClient.
+This package contains an isomorphic SDK for GraphRbacManagementClient to work with the Azure Active Directory Graph API which is now deprecated. Starting June 30th 2022, we will end support for Azure AD Graph and will no longer provide technical support or security updates. Apps using Azure AD Graph after this time will no longer receive responses from the Azure Active Directory Graph endpoint.
```

Comment:
> we will end support for Azure AD Graph

> Apps using Azure AD Graph after this time will no longer receive responses from the Azure Active Directory Graph endpoint

Does "ending support" == "no longer sending responses"?

Reply:
In this case, yes. The endpoint is going to be decommissioned in favor of MS Graph.

Reply:
Then I am not sure why this is relevant if the endpoint itself is not functional:

> no longer provide technical support or security updates

Reply:
I copied that entire bit from https://techcommunity.microsoft.com/t5/azure-active-directory-identity/update-your-applications-to-use-microsoft-authentication-library/ba-p/1257363